manifests/variables.yml

```diff
@@ -21,6 +21,12 @@ large_vm_spot_bid_price: 0.05
 xlarge_vm_instance_type: m5.xlarge
 xlarge_vm_spot_bid_price: 0.1
 
+high_cpu_large_vm_instance_type: c5.large
+high_cpu_large_vm_spot_bid_price: 0.05
```

Comment:
Can you provide some evidence for this bid price?

Reply:
Ah hold on do you think it should approximately match the on-demand price or the 1y 100% upfront reversed price? Here it's based on the reserved price but it probably makes more sense to use the on-demand price (which is approx double).

The above, `m5.xlarge`, appears to be approx based on the reserved price (which is actually ~$0.13 according to what I'm looking at)

(Either way this is only used for dev instances...)

Reply:
Have added a comment, no idea if it's what we _should_ be doing but :shrug: